recipe/meta.yaml

```diff
@@ -1,34 +1,41 @@
 {% set name = "scikit-surprise" %}
 {% set version = "1.1.3" %}
-{% set sha256 = "56bfec410b21278807b7538acb7405db99b44937259dd329762cee7e667d4bcc" %}
+{% set numpy = "1.19.5" %}  # [py<=39 and not (osx and arm64)]
+{% set numpy = "1.21" %}  # [py<=39 and (osx and arm64)]
+{% set numpy = "1.21" %}  # [py>39 and py<311]
+{% set numpy = "1.23" %}  # [py>=311]
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 56bfec410b21278807b7538acb7405db99b44937259dd329762cee7e667d4bcc
 
 build:
-  number: 1
-  script: python setup.py install --single-version-externally-managed --record record.txt
-
+  number: 0
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

Comment:
Can you also use `--no-build-isolation` here?

Reply:
Added. Thanks!